accounts-db/src/transaction_results.rs

```diff
@@ -29,11 +29,13 @@ pub struct TransactionResults {
 /// is what should be checked to detect a successful transaction. This
 /// enum provides a convenience method `Self::was_executed_successfully` to
 /// make such checks hard to do incorrectly.
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, Deserialize, Serialize)]
 pub enum TransactionExecutionResult {
     Executed {
         details: TransactionExecutionDetails,
+        #[serde(skip)]
```

Comment:
can we potentially abstract some of this away to something else in rpc-client? would like to avoid changing too much of core lib if possible